content/product/product_management/okrs.md

```diff
@@ -0,0 +1,58 @@
+# Product and engineering OKRs
+
+## Philosophy
+
+### Product and engineering share OKRs
+
+Product and engineering have a shared goal to deliver a product that provides value to our customers, so we set shared OKRs.
+
+### OKRs are a contract
```

Comment:
As mentioned in standup, can we change this to something like "By setting objectives and key results, we are able to make our priorities visible across different departments and management levels at the company"?

Reply:
Actually - double checking, do we set ambitious OKRs or ones that are treated like commitments that we are expected to hit? I've heard them described as both recently.